src/components/Proposal/ProposalDetailsPage.tsx

```diff
@@ -198,6 +220,25 @@ class ProposalDetailsPage extends React.Component<IProps, IState> {
             <div className={css.description}>
               <ReactMarkdown source={proposal.description}
                 renderers={{link: (props: { href: string; children: React.ReactNode }) => {
+                  if (props.href) {
+                    const url = new URL(props.href);
+                    const videoId = this.parseYouTubeVideoIdFromUri(props.href);
+                    if (videoId) {
+                      const start = url.searchParams.get("t") || "0";
+
+                      return <iframe className={css.embeddedVideo} frameBorder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowFullScreen
+                        src={`${url.protocol}//www.youtube-nocookie.com/embed/${videoId}?start=${start}`}>
+                      </iframe>;
+                    }
+                    else {
```

Comment:
usually we put this on the same line as the } above

Reply:
fixed

Reply:
fixed